extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesConfig.java

```diff
@@ -95,6 +96,12 @@
     @ConfigItem
     Map<String, PortConfig> ports;
 
+    /**
+     * The number of desired pods
+     */
+    @ConfigItem
+    OptionalInt replicas;
```

Comment:
I think we should probably use:

```
    @ConfigItem(defaultValue = "1")
    Integer replicas;
```

It will make the default more visible.

Reply:
Okey sounds like a good idea. I will update the pull request

Reply:
Oh sorry, I missed this. Any reason to not use `Integer` instead of `OptionalInt`?

Reply:
Once this is addressed, we can merge

Reply:
Yeah, it's never empty now, if the user doesn't specify a value `1` will be used by Quarkus so there is no need to be defensive.